src/mss/MssHandler.js

```diff
@@ -106,9 +104,14 @@ function MssHandler(config) {
         mssFragmentProcessor.processMoof(e);
     }
 
+    function registerEvents() {
+        eventBus.on(Events.INIT_REQUESTED, onInitializationRequested, instance, EventBus.EVENT_PRIORITY_HIGH);
+        eventBus.on(MediaPlayerEvents.FRAGMENT_LOADING_COMPLETED, onSegmentMediaLoaded, instance, EventBus.EVENT_PRIORITY_HIGH);
+    }
+
     function reset() {
         eventBus.off(Events.INIT_REQUESTED, onInitializationRequested, this);
-        eventBus.off(Events.FRAGMENT_LOADING_COMPLETED, onSegmentMediaLoaded, this);
+        eventBus.off(MediaPlayerEvents.FRAGMENT_LOADING_COMPLETED, onSegmentMediaLoaded, this);
```

Comment:
i not agree with you this time. ;-) What I want to do is to register in the Mss module on MediaPlayerEvents. It's not to create new specific smooth events. In the same way, in the MetricsHandlersController class of the reporting module, MediaPlayerEvents.METRIC_ADDED is referenced...Are you agree? ;-)